web/src/containers/Matchup/MatchupDisplay/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+
+import { IAppState } from 'src/interfaces';
+
+import { default as MatchupDisplay, IStateProps} from './MatchupDisplay';
+
+const mapStateToProps = (state: IAppState): IStateProps => {
+    const { matchup: matchupState } = state;
+    if(!matchupState) {
+        return {
+            team1: '',
+            team2: '',
+            matchup: null,
+        }
+    }
+    const { team1, team2, matchup } = matchupState;
+    return {
+        team1,
+        team2,
+        matchup,
+    }
+};
+
+export default connect(mapStateToProps)(MatchupDisplay as any); // TODO: typescript better
```

Comment:
I've been using this workaround which actually feels ok https://github.com/Microsoft/TypeScript/issues/25103#issuecomment-412806226

Reply:
Having said that, I'm not getting the error any more. Possibly fixed in TS?

Reply:
Yeah looks like this just works now, 🤷‍♂ I had been copy pasting the boilerplate for new components 😳